### ISSUE TITLE
[A11Y] Input markup is lacking

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1070]
## Finding

Related to #1068 

The form field is not marked up as being required (visually or in code). There is no (connected) label. Error is not announced. Error messages are not connected to their input field. Input field is not marked up as invalid.

## Location

login.eduid.nl, login and creation

<img width="604" height="353" alt="" src="https://github.com/user-attachments/assets/0e33490f-83ce-4564-9f6e-b5f5032b05f1" />

## Problems and solutions

In combination with the suggestions in #1068.

Add a `required`-attribute to the input.  Add a `<label>` with a `for`-attribute. Add a `role="alert"` to the error message. Connect the error message using `aria-describedby`. Add `aria-invalid` to a field with an error.

## Resources

- https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html 
- https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html
- https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
- https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-invalid
- https://github.com/OpenConext/OpenConext-myconext/issues/1068
